main.go

```diff
@@ -296,6 +344,8 @@ func main() {
 		return katsuRequestBasic("/api/public_search_fields", c)
 	})
 
+	e.GET("/wesruns", wesRequestWithDetailsAndPublic)
```

Comment:
i would prefer if this was formatted like `/wes-runs` (with matching front-end change ofc)